apps/condo/domains/common/schema/fields.js

```diff
@@ -2,6 +2,7 @@ const { hasValidJsonStructure } = require('@condo/domains/common/utils/validatio
 const { Integer } = require('@keystonejs/fields')
 const { Json } = require('@core/keystone/fields')
 const { JSON_UNKNOWN_VERSION_ERROR, REQUIRED_NO_VALUE_ERROR, JSON_EXPECT_OBJECT_ERROR } = require('@condo/domains/common/constants/errors')
+const { ADDRESS_META_FIELD_GRAPHQL_TYPES } = require('../../property/schema/fields/AddressMetaField')
```

Comment:
cross-domain imports should start with `@condo`

Reply:
Fixed in `34b72266`